src/Fields/BlueprintRepository.php

```diff
@@ -299,7 +299,8 @@ protected function filesIn($namespace)
 
             $files = File::withAbsolutePaths()
                 ->getFilesByType($directory, 'yaml')
-                ->mapWithKeys(fn ($path) => [Str::after($path, $directory.'/') => $path]);
+                ->mapWithKeys(fn ($path) => [Str::after($path, $directory.'/') => $path])
+                ->reject(fn ($path) => Str::endsWith($path, 'settings.yaml'));
```

Comment:
Shouldn't it check for ONLY that file name? In case someone has a blueprint called `duncan_settings.yaml`?